examples/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+SET(${PROJECT_NAME}_EXAMPLES
+  i-inverse-kinematics
+  overview-simple
+  #overview-urdf
+  )
+
+FOREACH(EXAMPLE ${${PROJECT_NAME}_EXAMPLES})
+  IF(BUILD_PYTHON_INTERFACE)
+    ADD_PYTHON_UNIT_TEST("py-example-${EXAMPLE}" "examples/${EXAMPLE}.py" "bindings/python")
+  ENDIF(BUILD_PYTHON_INTERFACE)
+  ADD_UNIT_TEST("example-${EXAMPLE}" ${EXAMPLE})
+  PKG_CONFIG_USE_DEPENDENCY("example-${EXAMPLE}" eigen3)
+  PKG_CONFIG_USE_DEPENDENCY("example-${EXAMPLE}" hpp-fcl)
```

Comment:
You should add a flag to check if HPP_FCL has been found

Reply:
done in 924953ee0a71aed189a5c158de52a2a375f0d8d3